utils/reviewsUtils.js

```diff
@@ -72,3 +73,11 @@ export async function getAverageRating(adapter, id) {
   }
   return { rating: averageRating, maxRating: maxRating };
 }
+
+export async function validateReview(review) {
```

Comment:
You have created this function as an `async` function, but it does no asynchronous work. There can be reasons to do something like this, but I'm guessing it's a mistake.